keyboards/tada68/keymaps/peippo/keymap.c

```diff
@@ -0,0 +1,52 @@
+#include QMK_KEYBOARD_H
+
+#define _BL 0
```

Comment:
You don't need to change this, but if you add more layers, you may want to switch to using an enum here, as that is easier to manage long term. 

This change doesn't have to be made now or even in the new future.   I mention it more as a suggestion for the future.